### PR TITLE
gtk+3: update to 3.24.41.

### PR DESCRIPTION
**Pull request by ahesford**
Also pulled an upstream patch that fixes resizing behavior for partially tiled Wayland clients.

#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

**Comment by ahesford** [comment]
By request from @sgn, the `gtk-update-icon-cache` tool is now a subpackage of `gtk+3` rather than `gtk+`.

File: srcpkgs/gtk+3/template [hunk 1]
```diff
@@ -17,8 +17,8 @@ configure_args="
  -Dexamples=false
  "
 conf_files="/etc/gtk-3.0/im-multipress.conf"
-hostmakedepends="gettext-devel glib-devel gtk-doc gtk-update-icon-cache perl
- pkg-config $(vopt_if wayland 'wayland-devel wayland-protocols')"
+hostmakedepends="gettext-devel glib-devel gtk-doc perl pkg-config
```
Comment (sgn) [inline srcpkgs/gtk+3/template:20]:
```suggestion
hostmakedepends="gettext-devel glib-devel gtk-doc perl pkg-config gdk-pixbuf
```

gtk+3 needs `gdk-pixbuf-pixdata`

File: srcpkgs/gtk+3/template [hunk 2]
```diff
@@ -55,6 +55,8 @@ desc_option_cloudproviders="Enable integration with cloudproviders, such as Next
 build_options_default="colord cups gir broadway wayland x11"
 if [ -z "$CROSS_BUILD" ]; then
 	build_options_default+=" gtk_doc"
+else
+	hostmakedepends+=" gtk-update-icon-cache"
 fi
```
Comment (sgn) [inline srcpkgs/gtk+3/template:61]: This change can eliminate the `hostmakedepends`
```suggestion
fi

pre_configure() {
	ln -sf /bin/false "${XBPS_WRAPPERDIR}/gtk-update-icon-cache"
}

```

File: srcpkgs/gtk+/template [hunk 3]
```diff
@@ -28,7 +31,7 @@ build_options="gir cups"
 build_options_default="gir cups"
 
 if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" automake libtool gtk-doc glib-devel gdk-pixbuf-devel gtk-update-icon-cache"
+	hostmakedepends+=" automake libtool gtk-doc glib-devel gdk-pixbuf-devel"
 fi
 
 pre_configure() {
```
Comment (sgn) [inline srcpkgs/gtk+/template:37]: With this change, `gtk-update-icon-cache` is not needed.
```suggestion
pre_configure() {
	ln -sf /bin/false "${XBPS_WRAPPERDIR}/gtk-update-icon-cache"
```

**Pull request merged**